### DISCUSSION TITLE
Using GenAI to Help Pick Your D & D Class

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#21] [Comments]
# Using GenAI to Help Pick Your D & D Class

Father, husband, developer relations and web standards expert, and cat demo builder.

https://www.raymondcamden.com/2024/04/11/using-genai-to-help-pick-your-d--d-class

**Comment by robertz** [comment]
Good stuff Ray. Now if you can just bake AI into a character trainer....
